ts/boot/configureInstabug.ts

```diff
@@ -28,7 +27,7 @@ export const initialiseInstabug = () => {
   Instabug.startWithToken(instabugToken, [Instabug.invocationEvent.none]);
 
   // Set primary color for iOS. The Android's counterpart is inside MainApplication.java
-  Instabug.setPrimaryColor(processColor(variables.contentPrimaryBackground));
+  Instabug.setPrimaryColor(variables.instabugColorInt);
```

Comment:
This is an Instabug bug. The static method signature accepts a number as a parameter. This is wrong. The right type is a string (e.g "#ff0000" set red color). So you have to patch setPrimaryColor to accept a string instead a number and, of course, you don't need instabugColorInt anymore

Reply:
Yes, with patch works correctly